testing/wasm-tests/tests/wasm.rs

```diff
@@ -22,3 +38,29 @@ async fn wasm_ws_transport_works() {
     let chain = client.rpc().system_chain().await.unwrap();
     assert_eq!(&chain, "Development");
 }
+
+#[wasm_bindgen_test]
+async fn light_client_works() {
```

Comment:
I guess that by enabling the "unstable-light-client" feature on this crate, we no longer are testing the "wasm but not via light client" route; maybe we should have two crates? `wasm-rpc-test` and `wasm-lightclient-test`? What do you think?

Reply:
Yep, that makes a lot of sense, have split up the testing into separate crates! Thanks for pointing out the light-client RPC!